src/shared/types/thoas/metadata.ts

```diff
@@ -21,8 +21,8 @@ export type ValueSetMetadata = {
 };
 
 export type XrefMetadata = {
```

Comment:
Are we sure the Metadata only applies to gene? could we have Metadata for transcript?

Reply:
There is metadata for a transcript (see a [CDM doc](https://github.com/Ensembl/ensembl-cdm-docs/blob/master/src/docs/feature_metadata.md)); however, I don't believe there is a name metadata for a transcript. Transcripts don't have names. Don't know how other features will shape up.

It's probably safe to just assign this particular shape specifically to gene name metadata since there name metadata doesn't exist for any other feature as of now.

Reply:
Agreed, the metadata field is used a lot of places and it can get confusing. It makes sense to have this one name geneNameMetadata.

Reply:
That's done